simulations/vip-373/arbitrumsepolia.ts

```diff
@@ -0,0 +1,19 @@
+import { expectEvents } from "src/utils";
+import { forking, pretendExecutingVip, testForkedNetworkVipCommands } from "src/vip-framework";
+
+import vip014 from "../../multisig/proposals/arbitrumsepolia/vip-014";
+import vip373 from "../../vips/vip-373/bsctestnet";
+import ACCESS_CONTROL_MANAGER_ABI from "./abi/AccessControlManager.json";
+
+forking(86016842, async () => {
+  before(async () => {
+    await pretendExecutingVip(await vip014());
+  });
+
+  testForkedNetworkVipCommands("vip373", await vip373(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [ACCESS_CONTROL_MANAGER_ABI], ["PermissionGranted"], [84]);
```

Comment:
Can we also check for GrantPermissionsExecuted and RevokePermissionsExecuted events?